proto/commonfate/access/provisioner/v1alpha1/provisioner.proto

```diff
@@ -31,6 +31,10 @@ message ProvisionRequest {
 
   // The ID of the request.
   commonfate.entity.v1alpha1.EID access_request_id = 7;
+
+  commonfate.control.integration.v1alpha1.Config additional_integration = 8;
```

Comment:
we should make this `repeated`, otherwise we'll need to add new fields if we ever need more than two integration configs (hopefully we never need that, but I think flexibility will be better here)